controllers/kokumetricsconfig_controller_test.go

```diff
@@ -858,6 +859,26 @@ var _ = Describe("MetricsConfigController - CRD Handling", func() {
 					return refetched.Status.Upload.LastSuccessfulUploadTime.IsZero()
 				}, timeout, interval).Should(BeFalse())
 			})
+			It("old default url - should update status to new default url", func() {
+				Expect(setup()).Should(Succeed())
+
+				instCopy.Spec.APIURL = oldDefaultAPIURL
```

Comment:
```suggestion
				instCopy.Spec.APIURL = metricscfgv1beta1.OldDefaultAPIURL
```
nitpick: Let's use the actual value here instead of redefining the url.

Reply:
done.